csharp/ql/src/change-notes/2024-03-11-remove-stored-query-variants.md

```diff
@@ -1,5 +1,5 @@
 ---
 category: majorAnalysis
 ---
-* The `Stored` variants of some queries (`cs/stored-command-line-injection`, `cs/web/stored-xss`, `cs/stored-ldap-injection`, `cs/xml/stored-xpath-injection`) have been removed. If you were using these queries, their results can be restored by enabling the `stored` threat model in your threat model settings.
+* The `Stored` variants of some queries (`cs/stored-command-line-injection`, `cs/web/stored-xss`, `cs/stored-ldap-injection`, `cs/xml/stored-xpath-injection`) have been removed. If you were using these queries, their results can be restored by enabling the `local` threat model in your threat model settings.
```

Comment:
`local` -> `file` and `database`. If `local` is added, other local sources will be added as well. We can recommend as a part of the change note to use `local` instead.